DigitalLearningSolutions.Web/Controllers/Certificate/CertificateController.cs

```diff
@@ -0,0 +1,135 @@
+﻿using DigitalLearningSolutions.Data.DataServices;
+using DigitalLearningSolutions.Data.Models.Centres;
+using DigitalLearningSolutions.Data.Models.Certificates;
+using DigitalLearningSolutions.Data.Models.Common;
+using DigitalLearningSolutions.Data.Utilities;
+using DigitalLearningSolutions.Web.Controllers.TrackingSystem.Centre.Configuration;
+using DigitalLearningSolutions.Web.Helpers;
+using DigitalLearningSolutions.Web.Helpers.ExternalApis;
+using DigitalLearningSolutions.Web.Services;
+using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Configuration;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewEngines;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using Microsoft.FeatureManagement.Mvc;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DigitalLearningSolutions.Web.Controllers.Certificate
+{
+    [FeatureGate(FeatureFlags.RefactoredTrackingSystem)]
+    [Authorize(Policy = CustomPolicies.UserCentreAdmin)]
+    public class CertificateController : Controller
+    {
+        private readonly ICentresDataService centresDataService;
+        private readonly ILogger<ConfigurationController> logger;
+        private readonly IMapsApiHelper mapsApiHelper;
+        private readonly IImageResizeService imageResizeService;
+        private ICertificateService certificateService;
+
+        public CertificateController(
+           ICentresDataService centresDataService,
+           IMapsApiHelper mapsApiHelper,
+           ILogger<ConfigurationController> logger,
+           IImageResizeService imageResizeService,
+           ICertificateService certificateService
+       )
+        {
+            this.centresDataService = centresDataService;
+            this.mapsApiHelper = mapsApiHelper;
+            this.logger = logger;
+            this.imageResizeService = imageResizeService;
+            this.certificateService = certificateService;
+        }
+        [Route("preview:int")]
```

Comment:
Could we go for `[Route("Certificate/Preview")]`, please?

Reply:
@kevwhitt-hee - Implemented.